### PR TITLE
Passing MABL_API_KEY should be optional as specified in the docs

### PR DESCRIPTION
**Pull request by jbaldassari**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -15,16 +15,17 @@ async function run() {
...
-  if (!(await authenticateWithApiKey(apiKey, nodePath))) {
-    return;
+  if (apiKey) {
+    if (!(await authenticateWithApiKey(apiKey, nodePath))) {
+      return;
+    }
   }
 
   if (workspace) {
+    if (!apiKey) {
```
Comment (gcooney) [inline src/index.ts:25]: I think it is possible to configure a workspace without an api key or am I wrong?

Comment (gcooney) [reply src/index.ts:25]: nvm, I think we do.

**Review by gcooney — APPROVED** [review]
Looks good.  I also made some doc updates to reflect the changes and fix a couple doc issues.

File: src/index.ts (hunk 1 above)
Comment (jbaldassari) [reply src/index.ts:25]: Not sure.  doesn't it try to validate it?

**Pull request merged**